tests/integration/gitlab/test_service.py

```diff
@@ -80,3 +82,7 @@ def test_project_create_in_the_group(self):
             repo=name_of_the_repo, namespace=namespace_of_the_repo
         )
         assert project.gitlab_repo
+
+    def test_service_without_auth(self):
+        service = GitlabService()
+        assert service.gitlab_instance
```

Comment:
Can you make some real call so we know it really works? It does not touch anything now...

Reply:
Yes, I could... However I tested it with `master` and it fails with `401 Unauthorized` :D